openedx/core/djangoapps/credentials/management/commands/tests/test_notify_credentials.py

```diff
@@ -22,6 +22,7 @@
 
 
 @skip_unless_lms
+@mock.patch(NOTIFY_CREDENTIALS_TASK)
```

Comment:
Every test mocks this function, so I've moved it to a class level mock rather than at the test level.